attestation-service/docker/as-restful/Dockerfile

```diff
@@ -17,25 +18,35 @@ RUN if [ "${ARCH}" = "x86_64" ]; then curl -L https://download.01.org/intel-sgx/
     apt-get update && apt-get install -y libsgx-dcap-quote-verify-dev; fi
 
 # Build and Install RESTful attestation-service
-RUN cargo install --path attestation-service --bin restful-as --features restful-bin --locked
+RUN cargo install --path attestation-service --bin restful-as --features restful-bin,${VERIFIER} --locked && \
+    strip /usr/local/cargo/bin/restful-as
```

Comment:
why do we strip the binary?

Reply:
Yea to make the binary smaller as more layers are added to the dockerfile